packages/preview-middleware-client/test/unit/adp/init-dialogs.test.ts

```diff
@@ -18,15 +18,16 @@ describe('Dialogs', () => {
                 }
             });
             initDialogs(rtaMock);
-            expect(addMenuItemSpy.mock.calls.length).toBe(1);
+            expect(addMenuItemSpy.mock.calls.length).toBe(2);
```

Comment:
Could we use the more concise `expect(addMenuItemSpy).toBeCalledTimes(2);`

Reply:
Fixed in: https://github.com/SAP/open-ux-tools/pull/1275/commits/1aa4fddcaaf9dbee692fff79c11a7b1ad11dad22